### PR TITLE
Up resolution and add timer progress

### PR DESCRIPTION
**Pull request by zainafzal08**
Adds in a little timer to all games. By default this rendered at like 6 pixels so updated pixi to make use of as much screen resolution as it can.

This broke the rendering of the phone screen which was relying on no resolution multiplier so manually set it's size to fix.

### CODE REVIEW

**Comment by zainafzal08** [comment]
Also update the main menu colors to match intro sequence

File: src/minigames/minigame.ts [hunk 1]
```diff
@@ -16,17 +19,31 @@ export class Minigame {
...
+      CLOCK_RADIUS,
+    );
+    this.clock.fill();
   }
 
   async attach() {
     this.ticker = new Ticker();
     if (this.lifetime !== undefined) {
       this.ticker.add(() => void this.onTick());
+      this.app.stage.addChild(this.clock);
```
Comment (josephkimsungha) [inline src/minigames/minigame.ts:46]: nit: should we put this on `this.container`?

File: src/minigames/minigame.ts [hunk 2]
```diff
@@ -38,6 +55,10 @@ export class Minigame {
     this.ticker?.destroy();
     this.ticker = null;
     this.app.stage.removeChild(this.container);
+    if (this.lifetime !== undefined) {
```
Comment (josephkimsungha) [inline src/minigames/minigame.ts:58]: Then we don't need this block

**Review by josephkimsungha — APPROVED** [review]

File: src/minigames/minigame.ts (hunk 1 above)
Comment (zainafzal08) [reply src/minigames/minigame.ts:46]: Prob but ceebs, this works.

**Pull request merged**